src/sentry/runner/initializer.py

```diff
@@ -402,13 +402,17 @@ def validate_options(settings):
 def __model_unpickle_compat(model_id, attrs=None, factory=None):
     from django import VERSION
 
+    if attrs is not None or factory is not None:
+        metrics.incr("django.pickle.loaded_19_pickle.__model_unpickle_compat")
+
     if VERSION[:2] in [(1, 10), (1, 11)]:
         return model_unpickle(model_id)
     else:
         raise NotImplementedError
 
 
 def __simple_class_factory_compat(model, attrs):
+    metrics.incr("django.pickle.loaded_19_pickle.__simple_class_factory_compat")
```

Comment:
I don't think this will work. I rebased `tmp/testing-django-1.10-1.11-pickled-models`, replaced this with an exit, and ran `unpickle_project_from_different_dj.py` to unpickle 1.9, 1.10 model pickles from 1.11, and everything passed.